design-proposals/network-binding-plugin/network-binding-plugin.md

```diff
@@ -257,6 +258,48 @@ VMI status and in one case to pass the data (processed) to the
 
 - DownwardAPI
 
+##### Device Info
+
+In some cases the multus status annotation
+contains [device-info](https://github.com/k8snetworkplumbingwg/device-info-spec/blob/main/SPEC.md).
+This info may be valuable for the binding plugin sidecar. For example
+`SRIOV` and `VDPA` plugins need this info.
+
+As the multus status annotation is not there yet on `virt-launcher` pod
+creation, the suggested solution is to use downwardAPI volume and mount
+it to the sidecar.
+The downwardAPI volume will track a new `virt-launcher` pod
+annotation `kubevirt.io/network-device-info-map` that will contain a
+mapping between the logical network name to the `device-info` of this
+network.
+Very similar to the
+existing [SRIOV downwardAPI](https://github.com/kubevirt/kubevirt/pull/8226)
+solution which is mapping the networkName to the `pci-address` of the
+device and mounting the downwardAPI volume to the `virt-launcher` pod.
+
+> **Note**: Old multus versions may not report the device-info.
```

Comment:
Please specify the version and what will happen if it is missing.

Reply:
I recall it was Multus that stated to reflect PCI info since version v3.7
https://github.com/kubevirt/community/blob/main/design-proposals/sriov-interfaces-mapping.md#example-1 
See the note in this section

Reply:
Thanks Or, updated the doc.